src/interactivity/inc/IHighDpiApi.hpp

```diff
@@ -22,14 +22,6 @@ namespace Microsoft::Console::Interactivity
         virtual BOOL SetProcessDpiAwarenessContext() = 0;
         [[nodiscard]] virtual HRESULT SetProcessPerMonitorDpiAwareness() = 0;
 
-        virtual ~IHighDpiApi() = 0;
-
-    protected:
-        IHighDpiApi() {}
-
-        IHighDpiApi(const IHighDpiApi&) = delete;
-        IHighDpiApi& operator=(const IHighDpiApi&) = delete;

```

Comment:
Why can we just delete these? Or is it redundant to delete them because they weren't declared in the first place?

Reply:
`= delete` actually explicitly un-declares and un-defines them even for types that inherit from it.
I'm not sure why we were doing this on an interface, so I removed it for simplicity's sake. IMO an interface shouldn't be able to dictate what its implementations can and can't do...